src/applications/appeals/testing/sc/content/livingSituation.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import readableList from 'platform/forms-system/src/js/utilities/data/readableList';
```

Comment:
Is this the right path for this import? It seems off to me that it's digging into the /src/ folder

Reply:
The full path is `src/platform/forms-system/src/js/utilities/data/readableList.js`; that's just how the [forms-system folder is set up](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/platform/forms-system/src/js) 🤷🏽